docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -106,6 +106,31 @@ i.e. you cannot bind a Horizontal Pod Autoscaler to a replication controller and
 The reason this doesn't work is that when rolling update creates a new replication controller,
 the Horizontal Pod Autoscaler will not be bound to the new replication controller.
 
+## Support for cooldown/delay
+
+When managing the scale of a group of replicas using the Horizontal Pod Autoscaler,
+it is possible that the number of replicas keeps fluctuating frequently due to
+dynamic nature of the metrics evaluated. This is sometimes referred to as "thrashing".
+
+Starting from v1.6, a cluster operator is enabled to mitigate this problem by tuning
```

Comment:
If "cluster operator" is a person, then you should just say "can" instead of "is enabled to", i.e.

"Starting from v1.6, a cluster operator can mitigate this problem..."

Reply:
Done